crates/miden-lib/asm/kernels/transaction/lib/account.masm

```diff
@@ -614,7 +614,7 @@ end
 #! - no map is found for ROOT.
 export.set_map_item.12
     # store index for later
-    dup loc_store.0
+    movup.2 loc_store.0
     # => [index, KEY, NEW_VALUE, OLD_ROOT, ...]
```

Comment:
There are some other places now where the OLD_ROOT is still on the stack in the comment which needs to be updated, e.g.:

```
    # => [index, KEY, NEW_VALUE, OLD_ROOT, ...]
```

should become:

```
    # => [index, KEY, NEW_VALUE, ...]
```